WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -174,7 +174,9 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
                 product = products.first { $0.productID == item.productID }
             }
             if product?.virtual == false || productVariation?.virtual == false {
-                tempTotalWeight += Double(productVariation?.weight ?? product?.weight ?? "0") ?? 0
+                for _ in 0..<item.quantity.intValue {
```

Comment:
Clever! Did you use a `for` loop here to avoid having to convert `item.quantity.intValue` to a `Double` to multiple it with the product weight? Or for readability? (Just asking for my own curiosity / learning.)

Reply:
I used this approach because we are using the `item.quantity.intValue` also for calculating the number of items rows, so this means that we will expect to have the same identical result here (which may not happen by multiplying a decimal number). Plus, we have in plan to change it in the near future because of this new issue: https://github.com/woocommerce/woocommerce-ios/issues/4640

Reply:
Ah, that makes sense! Thanks for explaining.